Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -445,4 +445,23 @@ class BrowserCoordinator: BaseCoordinator,
             findAndHandle(route: savedRoute)
         }
     }
+
+    func showTabTray() {
+        if let tabTrayCoordinator = childCoordinators[TabTrayCoordinator.self] {
+            tabTrayCoordinator.start()
+            (tabTrayCoordinator.router.navigationController as? UINavigationController).map { router.present($0) }
+        } else {
+            let navigationController = DismissableNavigationViewController()
+            navigationController.modalPresentationStyle = .formSheet
```

Comment:
The tab tray navigation controller was a `ThemedDefaultNavigationController` in previous code, should we have that one here too? 

Also, did it had a modalPresentationStyle set? I am not sure

Reply:
It's presented the same as the `LibraryViewController`

Reply:
`ThemedDefaultNavigationController` inherits from `DismissableNavigationViewController` add Themeable protocol to `DismissableNavigationViewController`. The problem is we need the root viewController to create `ThemedDefaultNavigationController` and with the current flow we create the navigation controller and then create the Coordinator and ViewController